### PR TITLE
Add sub-command to generate addresses for a chain

### PR DESCRIPTION
**Pull request by turbolent**
This is useful when using `util` with `execution-state-extract` with `--extract-payloads-by-address`

### CODE REVIEW

File: cmd/util/cmd/addresses/cmd.go [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+	"github.com/rs/zerolog/log"
+	"github.com/spf13/cobra"
+
+	"github.com/onflow/flow-go/model/flow"
+)
+
+var (
+	flagChain          string
+	flagCount          int
+	flagCommaSeparated bool
```
Comment (janezpodhostnik) [inline cmd/util/cmd/addresses/cmd.go:13]: Suggestion: 

`flagCommaSeparated` ->  `flagSeparator`

so user can specify any separator.

**Review by janezpodhostnik — APPROVED** [review]
This is great! Thank you

**Review by zhangchiqing — APPROVED** [review]

File: cmd/util/cmd/addresses/cmd.go (hunk 1 above)
Comment (turbolent) [reply cmd/util/cmd/addresses/cmd.go]: Good idea! Changed in d489273c82cb0b424c9372de1859464fe7bf44f7

File: cmd/util/cmd/addresses/cmd.go [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+	_ = Cmd.MarkFlagRequired("count")
+
+	Cmd.Flags().BoolVar(&flagCommaSeparated, "commas", false, "Separate using commas instead of newlines")
+}
+
+func run(*cobra.Command, []string) {
+	chain := flow.ChainID(flagChain).Chain()
+
+	generator := chain.NewAddressGenerator()
+
```
[the remark this answers is not in the document]
Comment (turbolent) [reply cmd/util/cmd/addresses/cmd.go:36]: This is unnecessary, as in that case, the for-loop will never execute the body

**Pull request merged**